ads/google/a4a/utils.js

```diff
@@ -172,6 +176,12 @@ export function googleAdUrl(
     if (isCanary(win)) {
       queryParams.push({name: 'isc', value: '1'});
     }
+    let eids = adElement.getAttribute('data-experiment-id');
+    if (opt_experimentIds) {
+      opt_experimentIds.forEach(experimentId => {
```

Comment:
Why not just pass array of experimentIds into mergeExperimentIds as opposed to a loop the splits/joins each time?